web/libs/editor/src/components/Timeline/Timeline.tsx

```diff
@@ -18,7 +18,8 @@ const TimelineComponent: FC<TimelineProps> = ({
   length = 1024,
   position = 1,
   framerate = 24,
-  hopSize = 1,
+  altHopSize = 1,
+  hopSize = altHopSize,
```

Comment:
the only place simple `hopSize` is used is AudioNext:100 and we should not use it anymore. what do you think about unifying all these params to only one?

Reply:
ahh good to know - but that feels like it should be done as part of a separate ticket to remove AudioNext altogether - we've had multiple Audio versions for too long and its time to unify